vcenter/vcenter.py

```diff
@@ -527,17 +530,24 @@ def get_file_bytes_from_datastore(self, remote_path_to_file, datastore_name):
 
         for i in range(3):
             try:
-                resp = requests.get(url=url, verify=False, headers={'Cookie': self._connection_cookie})
+                # resp = requests.get(url=url, verify=False, headers={'Cookie': self._connection_cookie})
+                resp = requests.get(
```

Comment:
I would add here comment that Cookie was dropped, because it resulted in less stability.

Reply:
ok